hack/tests/e2e-ansible.sh

```diff
@@ -238,6 +239,9 @@ add_go_mod_replace "github.com/operator-framework/operator-sdk" "$ROOTDIR"
 header_text "Build the project to resolve dependency versions in the modfile."
 go build ./...
 
+./hack/image/build-ansible-image.sh $(ANSIBLE_BASE_IMAGE):dev
```

Comment:
Is this necessary? Running the ansible e2e make targets should build the ansible image as a prerequisite. See [the ansible e2e make targets](https://github.com/operator-framework/operator-sdk/blob/5ec43a7d2aeeae59b158857825867c3c5b705d96/Makefile#L269-L273).

Reply:
Yeah, these modifications are not needed and they do break e2e tests. Will make changes while building base image instead to place the binary in the right place.